FirebaseRemoteConfig/Sources/RCNConfigExperiment.m

```diff
@@ -157,6 +179,18 @@ - (void)updateExperimentStartTime {
                          completionHandler:nil];
 }
 
+- (void)updateActiveExperiments {
```

Comment:
Nit: Might call this `updateActiveExperimentsInDb` or something to make it clear this is not related to updating active experiments in the Analytics SDK (I think those methods have similar names)

Reply:
Updated